lib/run_loop/simctl.rb

```diff
@@ -207,6 +207,49 @@ def wait_for_shutdown(device, timeout, delay)
       in_state
     end
 
+    def boot(device)
```

Comment:
Maybe rename to `start_boot` to indicate this method is async? You could also note that calling `RunLoop::CoreSimulator.wait_for_simulator_state(device, "Booted")` to wait for it is the thing to do.